open_dread_rando/files/levels/s050_forest.lc.lua

```diff
@@ -324,6 +324,10 @@ function s050_forest.OnSubAreaChange(old_subarea, old_actorgroup, new_subarea, n
       end
     end
   end
+  
+  if old_subarea == "collision_camera_036" and new_subarea == "collision_camera_026" and FOREST_POSTXRELEASE_APPLIED == false then
```

Comment:
I prefer to use the `QUARENTINE_OPENED` blackboard prop rather than the region specific ones